__tests__/app.test.js

```diff
@@ -72,6 +72,70 @@ describe('/api/reviews/:review_id', () => {
             })
         })
     })
+
+    test('PATCH - 404: Review ID not in bounds', () => {
+        const newVote = {
+            inc_votes: 5
+        }
+        return request(app)
+        .patch('/api/reviews/100')
+        .send(newVote)
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Review not found!')
+        })
+    });
+
+    test('PATCH - 400: Bad request', () => {
+        const newVote = {
+            inc_votes: 5
+        }
+        return request(app)
+        .patch('/api/reviews/bad_request')
+        .send(newVote)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Bad request!')
+        })
+    });
+
+    test('PATCH - 400: Incomplete Key', () => {
+        const newVote = {
+            inc: 5
+        }
+        return request(app)
+        .patch('/api/reviews/1')
+        .send(newVote)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Incomplete Object!')
+        })
+    });
+
+    test('PATCH - 200: Update votes and return the updated review', () => {
```

Comment:
I would recommend moving this test to the top as you should be working on the happy path first and then error handling after.